home/src/main/resources/rdf/tbox/firsttime/UILabelsVocabulary.ttl

```diff
@@ -1,39 +1,39 @@
 @prefix owl:   <http://www.w3.org/2002/07/owl#> .
 @prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-@prefix prop-data: <http://vivoweb.org/ontology/core/properties/individual#> .
-@prefix prop:  <http://vivoweb.org/ontology/core/properties/vocabulary#> .
+@prefix prop-data: <http://vivoweb.org/ontology/vitro-app/ui-label/individual#> .
+@prefix prop:  <http://vivoweb.org/ontology/vitro-app/ui-label/vocabulary#> .
```

Comment:
in other files uil and uil-data is used as a short prefix. Any reason why we should name that differently here? By the way prop-data (or uil-data) is not used in this file.

Reply:
Thank you for catching that.  Not only was the prefix different in that file, but so was the namespace itself.  Updated and removed unnecessary prop-data prefix.